qcelemental/models/results.py

```diff
@@ -37,8 +37,36 @@ class ResultProperties(BaseModel):
     mp2_opposite_spin_correlation_energy: float = None
     mp2_singles_energy: float = None
     mp2_doubles_energy: float = None
-    mp2_total_correlation_energy: float = None
+    mp2_correlation_energy: float = None
     mp2_total_energy: float = None
+    mp2_dipole_moment: List[float] = None
+
+    # CCSD Keywords
+    ccsd_same_spin_correlation_energy: float = None
+    ccsd_opposite_spin_correlation_energy: float = None
+    ccsd_singles_energy: float = None
+    ccsd_doubles_energy: float = None
+    ccsd_correlation_energy: float = None
+    ccsd_total_energy: float = None
+    ccsd_dipole_moment: List[float] = None
+    ccsd_iterations: int = None
+
+    # CCSD(T) keywords
+    ccsd_prt_pr_correlation_energy: float = None
+    ccsd_prt_pr_total_energy: float = None
+    ccsd_prt_pr_dipole_moment: List[float] = None
+
+    # CCSDT keywords
+    ccsdt_correlation_energy: float = None
+    ccsdt_total_energy: float = None
+    ccsdt_dipole_moment: List[float] = None
+    ccsdt_iterations: int = None
```

Comment:
CCSDT(Q) last one (ugh).

Reply:
Is this for completions sake? Since it looks like there a number of other CC methods out there too.

Reply:
Yea, honestly scratching my head on this one a bit after seeing it in practice. Anything beyond CCSD(T) might be better served in `qcvars`. @loriab?

Reply:
just checked and CCSDT(Q) wasn't added to qcschema. I think it's ok to skip for now.

Reply:
Yeah it doesn't seem there are a lot of codes out there that support past CCSD(T). I guess if we are still on this topic I'd only advocate for the addition of CC-F12 methods.

Reply:
in my experience, [these](https://github.com/qcdb/qcdb/blob/master/qcdb/psivarrosetta.py#L158-L183) are what you want to collect for f12 methods from molpro. not sure what balance of work-with-this-and-learn-from-practice or add-em-now to suggest.

Reply:
oh, and one shouldn't actually be storing (T), (T*), (T**), as those can be built by equation